migtests/tests/oracle/sequences/env.sh

```diff
@@ -1,4 +1,4 @@
 export SOURCE_DB_TYPE="oracle"
 export SOURCE_DB_SCHEMA=${SOURCE_DB_SCHEMA:-"TEST_SCHEMA"}
 export TARGET_DB_NAME=${TARGET_DB_NAME:-"oracle_sequences_test"}
-
+export RUN_SNAPSHOT_VALIDATIONS=${RUN_SNAPSHOT_VALIDATIONS:-"false"}
```

Comment:
Instead of skipping the snapshot validations completely, can we add the condition to the validation script to run only selective tests if this env var is set? That will give us more flexibility.